app/models/tipline_message.rb

```diff
@@ -3,7 +3,6 @@ class TiplineMessage < ApplicationRecord
 
   belongs_to :team
 
```

Comment:
Please add a validation to be sure that state is one of `sent`, `delivered` or `received`.

Reply:
Done